yt/visualization/geo_plot_utils.py

```diff
@@ -115,3 +115,15 @@ def get_mpl_transform(mpl_proj) -> Optional[FunctionType]:
     if key is None:
         instantiated_func = None
     return instantiated_func
+
+
+def _geographic_bounds(transform) -> tuple:
+    xlims: tuple = transform.x_limits
+    ylims: tuple = transform.y_limits
+    return xlims + ylims
+
+
+def _check_geographic_bounds(bounds: tuple, transform) -> bool:
+    # returns True if any of the bounds match the exact transform bounds
+    transform_bounds = _geographic_bounds(transform)
+    return any(bounds[bid].d == transform_bounds[bid] for bid in range(4))
```

Comment:
This function doesn't need to know how long we expect this iterables to be
```suggestion
    return any(b.d == tb for b, tb in zip(bounds, transform_bounds))
```

Reply:
also, since this is a small and private helper function, inlining it, or defining it closer to where it's actually used, would help readability of the callsite

Reply:
ok, i'll just move it down to where it's actually used. don't remember why i thought a separate function would be clearer...